src/reaction_network/network.cpp

```diff
@@ -227,17 +241,22 @@ void Network::init()
             boost::make_iterator_range(boost::in_edges(reaction, m_graph))) {
           v_desc_t reactant = boost::source(ei_in, m_graph);
 
-          #if !defined(WCS_HAS_EXPRTK)
-          //check for the reactants which are not actually reactants and put their stoichiometry 0
-          if ( std::find(params_reactants.begin(), params_reactants.end(),
-          m_graph[reactant].get_label()) == params_reactants.end()) {
-            //std::cout << "Not reactant " << m_graph[reactant].get_label() << std::endl;
-            m_graph[ei_in].set_stoichiometry_ratio(0);
-          }
-          #endif // !defined(WCS_HAS_EXPRTK)
+          // #if !defined(WCS_HAS_EXPRTK)
```

Comment:
Is this temporary or permanent?

Reply:
This is temporary. We have done this only for the generated code. I commented this in order to have a fair comparison.